pkgs/build-support/compress-drv/web.nix

```diff
@@ -1,7 +1,8 @@
 {
-  zopfli,
   brotli,
   compressDrv,
+  lib,
```

Comment:
Putting `lib` in the middle just looks super odd to me. Nothing wrong with it, I guess.

Reply:
It's sorted alphabetically.